### DISCUSSION TITLE
Allow Administrator to Limit Requests for Users

### DISCUSSION THREAD

**Discussion by cosmosxcii** [sct/overseerr#1077] [Feature Requests]
A popular tool for server managers is to introduce limits as to how many movies/shows can be requested by users per week.

In addition to this, within the "Edit User" page, there could be an override which would give the server manager the functionality to granularly control the request limit for each user.

**Comment by TheCatLady** [comment]
@cosmosxcii This is being worked on already. You can follow the existing feature request here: https://github.com/sct/overseerr/issues/263
